src/satisfies/satisfies-strongly-typed-url.ts

```diff
@@ -0,0 +1,16 @@
+//Satisfies
+
+//The operator that gives you a a way to add type annotations to values without using type inference.
+
+//Example Strongly Typed URL
+type UrlParams = {
+  id: string;
+  name: string;
+};
+
+//Property 'name' is missing in type '{ id: string; }' but required in type 'UrlParams'.
+// const params = new URLSearchParams({
+//   id: '123',
+// } satisfies UrlParams);
```

Comment:
Based on the TS [documentation](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html) what I understand is that the `satisfies` operator is not used within expressions or function calls but rather for standalone type checking of a value.

So I believe this example is more accurate:
```suggestion
// Property 'name' is missing in type '{ id: string; }' but required in type 'UrlParams'.
// const params = { 
// id: '123',
// } satisfies UrlParams;

// [optional to match your previous example]: const params = new URLSearchParams(paramsObject);
```

Reply:
it's pretty much the same no need to adjust the code

It might be more readable if placed into a separate const following your suggestiong however in our case we would like to specify that the loose type safety on `URLSearchParams` constructor can have enforced types